### PR TITLE
README nits

### PR DESCRIPTION
**Pull request by chris-stytch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,58 +18,66 @@ Follow the steps below to get this application fully functional and running usin
...
-In your terminal clone the project and install dependencies
+In your terminal clone the project and install dependencies:
 ```bash
 git clone https://github.com/cal-stytch/test-stytch-react-example.git
 cd test-stytch-react-example
 npm i
 ```
 
-Next, create `.env` file by running the command below and your **public token**. Learn more about Create React App's support for [custom environment variables here](https://create-react-app.dev/docs/adding-custom-environment-variables/).
+Next, create `.env.local` file by running the command below and your `public_token`. Learn more about Create React App's support for [custom environment variables here](https://create-react-app.dev/docs/adding-custom-environment-variables/).
```
Comment (cal-stytch) [inline README.md:52]: Did you test this?

Comment (chris-stytch) [reply README.md:52]: Yup, still works. `.env.local` is in gitignore already, just suggested the change to align with existing example apps.

File: public/index.html [hunk 2]
```diff
@@ -14,14 +14,15 @@
...
-      <img src="logo.svg" />
+      <a class="header" href="/">
+        <img src="logo.svg" width="190px"/>
+      </a>
       <div class="link-container">
-        
-        <a class="header" target="_blank" href="https://www.stytch.com/docs">Stytch docs</a>
-        <a class="header" target="_blank" href="https://github.com/cal-stytch/test-stytch-react-example">
+        <a class="header" target="_blank" href="https://www.stytch.com/docs">Stytch Docs</a>
+        <a class="header" target="_blank" href="https://www.github.com">
```
Comment (chris-stytch) [inline public/index.html:25]: Remember to update this once its live.

File: public/index.html [hunk 3]
```diff
@@ -14,14 +14,15 @@
   </head>
   <body>
     <noscript>
-      The Stytch React example application requires Javascript to run.
+      The Stytch React example application requires JavaScript to run.
     </noscript>
     <header>
-      <img src="logo.svg" />
+      <a class="header" href="/">
+        <img src="logo.svg" width="190px"/>
```
Comment (chris-stytch) [inline public/index.html:21]: Resized this a bit to better match "Stytch Docs" header image. Made it link to "home".

**Review by cal-stytch — APPROVED** [review]

**Pull request merged**